packages/url/src/index.js

```diff
@@ -34,6 +34,51 @@ export function addQueryArgs( url, args ) {
 	return baseUrl + '?' + stringify( { ...query, ...args } );
 }
 
+/**
+ * Returns a single query argument of the url
+ *
+ * @param  {string} url URL
+ * @param  {string} arg Query arg name
+ *
+ * @return {Array|string} Query arg value.
+ */
+export function getQueryArg( url, arg ) {
+	const queryStringIndex = url.indexOf( '?' );
+	const query = queryStringIndex !== -1 ? parse( url.substr( queryStringIndex + 1 ) ) : {};
+
+	return query[ arg ];
+}
+
+/**
+ * Determines whether the URL contains a given query arg.
+ *
+ * @param  {string}    url URL
+ * @param  {...string} arg Query arg name
+ *
+ * @return {boolean} Whether or not the URL contains the query aeg.
+ */
+export function hasQueryArg( url, arg ) {
+	return typeof getQueryArg( url, arg ) !== 'undefined';
```

Comment:
FYI: Comparing to `undefined` directly should work just as well, and saves some characters. `typeof` would be useful only if trying to reference a variable which hasn't been initialized, or if we were targeting pre-ES5 browsers (IE7) where `undefined` could be overwritten.